python/paddle/v2/framework/tests/test_image_classification_train.py

```diff
@@ -123,11 +240,11 @@ def conv_block(input,
                        fetch_list=[avg_cost])
 
         loss = np.array(outs[0])
-        # print("pass_id:" + str(pass_id) + " batch_id:" + str(batch_id) +
-        #       " loss:" + str(loss))
+        print("pass_id:" + str(pass_id) + " batch_id:" + str(batch_id) +
+              " loss:" + str(loss))
         batch_id = batch_id + 1
 
-        if batch_id > 1:
+        if batch_id > 10:
```

Comment:
The following comment tells that two minibatches is sufficient to make the test conclusion. Should we change the comment as well?

Reply:
reverted to 2, 10 is for self-test.